administrative-sdk.js

```diff
@@ -573,7 +583,7 @@ class Sdk {
     var self = this;
     connection.onerror = function(e) {
       console.log('WebSocket error: ' + e);
-      self.fireEvent('websocketError', [e]);
+      self.fireEvent('websocketError');
```

Comment:
I don't think this change is related. Or at least, it shouldn't be. Please revert this line.